apps/sms/views/shared/js/activity_handler.js

```diff
@@ -318,8 +318,9 @@ var ActivityHandler = {
             data: { id, threadId }
           };
 
-          return titlePromise.then((title) => {
-            var notification = new Notification(title, options);
+          return titlePromise.then((title) =>
+            NotificationHelper.send(title, options)
```

Comment:
Using the NotificationHelper.send here can set the title with id/args directly instead of another formatValue for l10n string.